internal/certs/certs.go

```diff
@@ -172,6 +178,15 @@ func New(isCA bool, issuer *Issuer, opts ...Option) (*Certificate, error) {
 		} else {
 			template.Subject.CommonName = "intermediate elastic-package CA"
 		}
+		// If the requester is a client we set clientAuth instead
+	} else if isClient {
+		template.ExtKeyUsage = []x509.ExtKeyUsage{
+			x509.ExtKeyUsageClientAuth,
```

Comment:
We probably need to update `checkExpectedCertUsage` so it accepts certificates that are not CA, and that don't have `x509.ExtKeyUsageServerAuth` extension.
With this we should avoid regenerating the client certificates on every run.

Reply:
Yeah. Checked if either `x509.ExtKeyUsageServerAuth` or `x509.ExtKeyUsageClientAuth` is present